packages/@okta/vuepress-site/docs/guides/pwd-optional-widget-sign-in-email/main/index.md

```diff
@@ -57,9 +61,13 @@ The Sign-In Widget displays a page for the user to start verifying their identit
 
 The user clicks **Send me an email** to begin the email challenge flow.
 
-<div class="half">
+<div class="half border">
+
+![The Sign-in Widget verify-email page with a Send me an email button.](/img/pwd-optional/pwd-optional-widget-send-email-page.png)
 
-![Screenshot showing the Sign-in Widget verify-email page with a Send me an email button.](/img/pwd-optional/pwd-optional-widget-send-email-page.png)
+<!--
+Source image: https://www.figma.com/file/YH5Zhzp66kGCglrXQUag2E/%F0%9F%93%8A-Updated-Diagrams-for-Dev-Docs?node-id=2188%3A1775 pwd-optional-widget-send-email-page
```

Comment:
Having second thoughts about mocking the Widget screens. Can we confirm this is okay with either Chris or Maurice?